packages/@sfx/core/test/unit/common/core.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect, sinon, spy, stub } from '../../utils';
+import Core from '../../../src/core';
+import * as CoreUtils from '../../../src/utils/core';
```

Comment:
I'd love to get `src/` on the path so that modules can simply start there. The above can cause issues if/when refactoring the directory structure needs to happen.

Reply:
I tried using TypeScript's `baseUrl` and `paths` compiler options, which enables [path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping), but it looks like [`tsc` does not output resolved paths](https://github.com/microsoft/TypeScript/issues/26722).

There are two packages that look like they may solve that issue (namely [tscpaths](https://github.com/joonhocho/tscpaths) and [ts-transformer-inputs](https://github.com/grrowl/ts-transformer-imports)), but they don't seem to be widely used, and one of them explicitly says that it's not ready for production. The oft-recommended [module-alias](https://github.com/ilearnio/module-alias) is not suitable for us since this is a library and not an application. Doing it through Webpack is not an option since we don't use Webpack to build the package, and we don't want a single bundle for the whole package.

As much as I would like path aliasing, I don't think we can have it under our current circumstances unless we decide that the risk of using either tscpaths or ts-transformer-inputs is worth it.